core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadService.java

```diff
@@ -22,6 +22,7 @@
 import android.webkit.URLUtil;
 
 import org.apache.commons.io.FileUtils;
+import org.shredzone.flattr4j.model.User;
```

Comment:
This change (import org.shredzone.flatter4j.model.User;) does not seem relevant to this fix / commit.